inspirehep/modules/migrator/cli.py

```diff
@@ -54,6 +57,8 @@
 def migrator():
     """Command related to migrating INSPIRE data."""
     logging.basicConfig()
+    # Disable auto-indexing receiver in migration tasks
+    models_committed.disconnect(receive_after_model_commit)
```

Comment:
I would expect this to leave it disconnected, so I'm not sure if this is correct, but honestly I have no idea...